### PR TITLE
ready for review: move npm package to scoped name @qooxdoo/framework

### PR DESCRIPTION
**Pull request by hkollmann**
See #9663

### CODE REVIEW

**Comment by cboulanger** [comment]
If we want to integrate qx.Server, don't we need to script something to achieve that (I assume that's currently done by the python scripts that you have removed).

**Comment by hkollmann** [comment]
There is missing the compilation of the qx library wich is used by qxcompiler.

**Comment by hkollmann** [comment]
This small changes allow to compile a server package with qooxdoo compiler and use this inside the compiler after some compiler changes: https://github.com/qooxdoo/qooxdoo-compiler/pull/373

**Review by oetiker — DISMISSED** [review]
Can we make it so that 'old users' of qooxdoo-sdk get notified in some way about the name change ?

**Comment by hkollmann** [comment]
We can deprecate the old npm packages.

File: compile.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "defaultTarget": "build",
+  "locales": [
+    "en"
+  ],
+  "libraries": [
+    "./framework"
+  ],
+  "applications": [
+    {
+      "class": "qx.application.Native",
```
Comment (cboulanger) [inline compile.json]: I wonder about the name "qx.application.Native" - wouldn't "qx.application.Server" be more fitting? What would "Native" mean in this context?

**Comment by cboulanger** [comment]
Great work, thanks so much. 

For everything to be consistent, we also need to update `Manifest.json/requires`, so that it is now "@qooxdoo/compiler" and "@qooxdoo/framework" instead of "qooxdoo-sdk" and "qooxdoo-compiler" - although the old values need to be accepted, too  so that code doesn't break.

**Comment by cboulanger** [comment]
We then should write a blog post to announce the latest changes.

File: compile.json (hunk 1 above)
Comment (hkollmann) [reply compile.json]: The class here is needed by the compiler and is just an dummy which will be loaded during startup. 
The native application is the app for non GUI apps and do nothing.

**Comment by hkollmann** [comment]
> Great work, thanks so much.
> For everything to be consistent, we also need to update Manifest.json/requires, so that it is now "@qooxdoo/compiler" and "@qooxdoo/framework" instead of "qooxdoo-sdk" and "qooxdoo-compiler" - although the old values need to be accepted, too  so that code doesn't break.
> Can you add that?

That's should be part of the qxcompiler changes. Yes i can add it there.

**Review by cboulanger — DISMISSED** [review]

File: .travis/deploy_npm [hunk 2]
```diff
@@ -38,16 +37,15 @@ exe npm -v
...
+npx qx compile -v
+
+
 # fill .npmrc with access token
 echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 # publish to npm
-exe npm publish 
-
-# build qooxdoo server package
-cd $BASE_DIR/component/standalone/server
```
Comment (johnspackman) [inline .travis/deploy_npm]: why are we removing this - has it been replaced?

Comment (hkollmann) [reply .travis/deploy_npm]: Yes. Server package becomes part of @qooxdoo/framework compiled by qx:


# get qooxdoo compiler, must be changed to @qooxdoo/compiler later 
npm install qxcompiler  
# compile framework 
npx qx compile -v

**Review by johnspackman — DISMISSED** [review]

**Review by level420 — DISMISSED** [review]

**Comment by cboulanger** [comment]
> We can deprecate the old npm packages.

Seems quite easy: https://docs.npmjs.com/cli/deprecate

**Review by oetiker — DISMISSED** [review]

**Review by derrell — CHANGES_REQUESTED** [review]
When there's only one, the translation file is typically not committed to git since it is autogenerated from the source when the compiler is run. There is nothing in a framework package that can be run by itself without running the compiler, right? Is there a reason that translation files (.po) need be included here in the package?

**Comment by hkollmann** [comment]
They have been there for the old toolchain.

**Comment by hkollmann** [comment]
Should i revert the changes from node toolchain?

**Comment by cboulanger** [comment]
Since the aren't any translations in the file, isn't this the equivalent of "no translation file exists"? Does the generator expect un untranslated .po file and throw if it is not there? Otherwise, I'd be in favor of not including it.

**Comment by hkollmann** [comment]
I can revert it - but i will not delete it. I do not know what is happening with generate.py if it‘s deleted and i am not longer working with it.

**Comment by hkollmann** [comment]
I reverted the en.po file to it‘s original version - so you must review again

**Review by derrell — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Comment by cboulanger** [comment]
How do we handle deprecation of the "qooxdoo-sdk" framework? As discussed in the compiler repo, we probably should create a branch off the latest master before merging this PR, and then modifying this branch to display the deprecation notice in its readme.md.

**Review by cboulanger — APPROVED** [review]

**Comment by hkollmann** [comment]
I will deprecate the npm package after @qooxdoo/framework is working. IMHO we do not need a branch in qooxdoo because we have only changes in manifest.json.

Von meinem iPad gesendet

**Comment by hkollmann** [comment]
Do you have a good name for the latest master?

**Comment by cboulanger** [comment]
@hkollmann I'd call it `v6_0_0_alpha` since the new package will (eventually) be ^beta - does that make sense?

**Comment by hkollmann** [comment]
Should be the last Legacy 5.0.x package. I thought master contains the 6.0.x version 

Von meinem iPhone gesendet

**Comment by hkollmann** [comment]
@cboulanger what´s about 5.0.x_legacy_branch ?

**Comment by cboulanger** [comment]
5 Approvals, enough days have passed, merging. Thank you @hkollmann

**Pull request merged**